app/ui/browser-modern/actions/page-effects.js

```diff
@@ -102,17 +102,19 @@ export function navigateCurrentPageForward() {
   };
 }
 
-export function navigatePageRefresh(pageId, doc = document) {
+export function navigatePageRefresh(pageId, ignoreCache, doc = document) {
```

Comment:
Nit: any reason why we're not just passing in all options here from `navigateCurrentPageRefresh`?